scenarios/bf1_basic_functionality.yaml

```diff
@@ -0,0 +1,263 @@
+version: 2
+
+settings:
+  gas_price: {{ gas_price }}
+  services:
+    pfs:
+      url: "{{ pfs_with_fee }}"
+    udc:
+      enable: true
+      token:
+        deposit: true
+        balance_per_node: "{{ ms_reward_with_margin + 200 * 2 * pfs_fee}}"
+
+token:
+  address: "{{ transfer_token }}"
+  balance_fund: 10_000_000_000_000_000_000
+
+nodes:
+  count: 5
+  raiden_version: local
+
+  default_options:
+    gas-price: fast
+    routing-mode: pfs
+    pathfinding-max-paths: 5
+    pathfinding-max-fee: "{{ pfs_fee }}"
+    enable-monitoring: true
+    flat-fee:
+      - "{{ transfer_token }}"
+      - 0
+    proportional-fee:
+      - "{{ transfer_token }}"
+      - 0
+    proportional-imbalance-fee:
+      - "{{ transfer_token }}"
+      - 0
+    default-reveal-timeout: {{ settlement_timeout_min // 4 }}
+  node_options:
+    0:
+      matrix-server: "{{ matrix_servers[0] }}"
+    1:
+      matrix-server: "{{ matrix_servers[1] }}"
+    2:
+      matrix-server: "{{ matrix_servers[2] }}"
+    3:
+      matrix-server: "{{ matrix_servers[3] }}"
```

Comment:
I think we have only 2 servers now, don't we? It'd be good to have this already adapted to work with the number of servers we have, so the SP server can already pick that up

Reply:
IIRC, the SP goes through the available number of servers multiple times if not enough servers are configured. If you actually got an error about it, let me know.